examples/with-typescript-graphql/pages/api/graphql.ts

```diff
@@ -1,12 +1,17 @@
-import { ApolloServer } from 'apollo-server-micro'
-import { schema } from '../../lib/schema'
+import { createServer } from '@graphql-yoga/node'
+import { readFileSync } from 'node:fs'
 
-const apolloServer = new ApolloServer({ schema })
+import resolvers from 'lib/resolvers'
 
-export const config = {
-  api: {
-    bodyParser: false,
+const typeDefs = readFileSync('lib/schema.graphql', 'utf8')
```

Comment:
worth to check this on a vercel deployment. readFileSync is highly dependent on cwd